### ISSUE TITLE
Change input for erf calculation

### ISSUE THREAD

**Issue by mcFloskel** [mcFloskel/houseNet#9]
The calcluation of the effective receptive field (erf) has to be changed in the following way:
* Change the input for gradient calculation to validation images
* Load the gradient input with a Keras Sequence
* Calculate the gradient in batches

**Comment by mcFloskel** [comment]
The input for the gradient can now be loaded with a Sequence class.
This can be seen in the updated example file where the DataLoader is used to load images from the validation data.

Currently the calculation is still done for each image separately. Batch calculation has to be added.
